package.json

```diff
@@ -22,19 +22,23 @@
     "raw-loader": "^0.5.1",
     "to-string-loader": "^1.1.4",
     "typescript": "~1.8.9",
-    "typings": "~1.0.3",
+    "typings": "~1.3.2",
```

Comment:
not mandatory, just a updated version for typings, I thought it'll be a best practice to keep things up to date, will revert